controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -313,7 +313,7 @@ void wfa_ca::handle_wfa_ca_message(
         }
 
         if (params.find("type") != params.end()) {
-            if (params["type"] != "test bed" && params["type"] != "dut") {
+            if (params["type"] != "test bed" && params["type"] != "DUT") {
```

Comment:
Basically, the CAPI commands should be **not** case sensitive with some exceptions like SSID, so maybe it was better to downcase the content of params["type"]. But on the other hand, it will be ugly and if the UCC sends it all the time up-case, then I guess it is ok.

Reply:
It sends it in uppercase